examples/index.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<iframe width="600px" height="800px" src="http://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
-<script src="../build/embed.js" defer></script>
+<iframe width="600px" height="800px" src="https://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
```

Comment:
You're not seeing stuff like this on your end?:
![screenshot 2015-01-26 22 34 32](https://cloud.githubusercontent.com/assets/674107/5914506/86ff7d74-a5ab-11e4-94f1-cb89de2d1d9d.png)